config/config.go

```diff
@@ -68,6 +77,10 @@ func DefaultConfig() *Config {
 		MintAddress:           "default",
 		ReissuanceAsset:       "asset-id-or-name",
 		ValidatorAddress:      "plmnt1w5dww335zhh98pzv783hqre355ck3u4w4hjxcx",
+		DistributionAddrInv:   "",
```

Comment:
Put some sane values in here to help the user. Maybe our three testnet liquid addresses?